package.json

```diff
@@ -12,11 +12,15 @@
     "lint": "eslint .",
     "lint:fix": "eslint . --fix"
   },
+  "types": "./dist/types.d.ts",
   "homepage": "https://turbopuffer.com",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/turbopuffer/turbopuffer-typescript.git"
   },
+  "browser": {
+    "./dist/httpClient/node.js": false
+  },
```

Comment:
Majority of usecases of putting tpuf in the browser would be through some sort of library/framework (React, Next, etc.) and if not using a framework, a simple bundler (webpack, browserify, etc.) would definitely be needed to use the npm package. This field prevents the httpClient/node.js from being loaded so it won't run into issues with Node-specific imports